src/main/java/uk/gov/ons/census/fwmtadapter/messaging/CaseUpdatedReceiver.java

```diff
@@ -85,6 +86,9 @@ private void buildAndSendActionInstruction(
       actionInstruction.setSecureEstablishment(caze.getMetadata().getSecureEstablishment());
     }
 
+    actionInstruction.setBlankFormReturned(
+        event.getPayload().getMetadata().getBlankQuestionnaireReceived());
```

Comment:
this will cause a null pointer exception when the metadata is null... look at the line above

Reply:
`canIgnoreEvent()` checks to see if the metadata is null before it reaches the switch case statements so I don't think it will

Reply:
in which case, would you remind getting rid of the if statement above

Reply:
my bad... different metadatas